src/sst_file_writer.rs

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 Lucjan Suski
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//`
+
+use crate::{ffi, ffi_util::to_cpath, Error, Options};
+
+use libc::{self, c_char, size_t};
+use std::ffi::CString;
+use std::marker::PhantomData;
+use std::path::Path;
```

Comment:
Make `std` imports nested please.

Reply:
from what i've seen you never do that in other files, can you show the desired effect?

Reply:
If [this](https://github.com/rust-rocksdb/rust-rocksdb/pull/421/commits/2ff1d0d2e6fa0ec6baeeb1e3d66e4ba3d89c8bc6) is what you meant, I don't think it makes sense to do that only for this particular file

Reply:
Yes. That's exactly what I mean. I'll change it in other places later.